api/model/src/main/resources/META-INF/openapi.yaml

```diff
@@ -57,30 +57,42 @@ components:
       value: "main"
 
     refWithHash:
-      value: "main@2e1cfa82b035c26cbbbdae632cea070514eb8b773f616aaeaf668e2f0be8f10d"
+      value: "main@1122334455667788112233445566778811223344556677881122334455667788"
+
+    refWithTimestampMillisSinceEpoch:
+      value: "main*1685185847230"
+
+    refWithTimestampInstant:
+      value: "main*2021-04-07T14:42:25.534748Z"
+
+    refWithNthPredecessor:
+      value: "main~10"
+
+    refWithMergeParent:
+      value: "main@1122334455667788112233445566778811223344556677881122334455667788^2"
 
     refDefault:
       value: "-"
 
     refDetached:
-      value: "@2e1cfa82b035c26cbbbdae632cea070514eb8b773f616aaeaf668e2f0be8f10d"
+      value: "@1122334455667788112233445566778811223344556677881122334455667788"
 
     refForDiffWithHash:
-      value: "main*2e1cfa82b035c26cbbbdae632cea070514eb8b773f616aaeaf668e2f0be8f10d"
+      value: "main*1122334455667788112233445566778811223344556677881122334455667788"
```

Comment:
Should this example actually use `@` instead of `*`? The old text is also invalid - it might have been a leftover from older API v2 experiments :facepalm: Could you fix it since we're changing it anyway?

By the way, I think `2e1cfa82b035c26cbbbdae632cea070514eb8b773f616aaeaf668e2f0be8f10d` is fine for examples, unless we're using parent references with it.

Reply:
updates